client/components/payment-activity/style.scss

```diff
@@ -142,10 +140,48 @@
 			@include amount-styles;
 		}
 
+		&__tooltip {
+			.wcpay-inline-notice.components-notice {
+				margin-top: 8px;
+				margin-right: 0;
+				padding: 8px;
+				color: $studio-blue-70;
+
+				.components-notice__content {
+					font-size: 11px;
+					line-height: 16px;
+					white-space: pre-wrap;
+					margin: 0;
+				}
+			}
+		}
+	}
+
+	#wcpay-payment-activity-data {
+		&__total-payment-volume {
+			@include breakpoint( '<660px' ) {
+				border-bottom: 1px solid $gray-200;
+				padding-top: 0;
+				padding-bottom: 24px;
+			}
+		}
+	}
+
+	.wcpay-tooltip__tooltip {
+		font-size: 13px;
+		line-height: 20px;
+		width: 294px;
+		top: 102px;
+		left: -16px;
+	}
+
+	.wcpay-tooltip__tooltip.wcpay-payment-activity-data__total-payment-volume__tooltip {
+		top: 178px;
+		left: -10px;
+
 		@include breakpoint( '<660px' ) {
-			border-bottom: 1px solid $gray-200;
-			padding-top: 0;
-			padding-bottom: 24px;
+			top: 50px;
+			left: 80px;
 		}
```

Comment:
These are awesome suggestions, Thanks @Jinksi ! 

I have pruned away the extra overrides. Most of them did some subtle differences the Figma design had over what's there for standard tooltips. You are right, if there is a larger design change needed, it is better to do it on the tooltip itself rather than placing overrides. 

I have removed the unwanted overrides here - https://github.com/Automattic/woocommerce-payments/pull/8659/commits/cd916465d247afd626a6546ed8564d8c94020c98

I added the maxWidth prop instead of CSS property - https://github.com/Automattic/woocommerce-payments/pull/8659/commits/dc55a058cd68856b6bb7d83f35d73ef9e7492b0c. I had clearly missed this. Thanks for showing it to me!

<a href="https://github.com/Automattic/woocommerce-payments/assets/4162931/33f344c5-2b0f-42f1-9c3e-1fec8fdae757"><img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/33f344c5-2b0f-42f1-9c3e-1fec8fdae757+" width="550px" /></a>

This seems to be working strangely different from the tooltip on the `Total balances` tile in the previous widget. 

I tried grokking the code within TooltipBase that calculates the left margin, but was unable to find a good way to change this. For now I have used an override to fix the tooltip at the required 45px margin - aaf0cd6 

I will be glad to know the best way of doing this and revisit.